lib/components/OverflowMenuItem/OverflowMenuItem.docs.tsx

```diff
@@ -27,6 +27,28 @@ const docs: ComponentDocs = {
         </Box>
       ),
     },
+    {
```

Comment:
I wouldn't add a new example for this as it doesn't test anything and isn't really adding anything to the docs.